### PR TITLE
refactor(chore): add root linter

### PR DESCRIPTION
**Pull request by UnbearableBear**

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+root: true
+extends:
+  - "@socialgouv/eslint-config-recommended"
+
+rules:
+  prefer-const: error
+  no-console: off
```
Comment (lionelB) [inline .eslintrc.yml]: we don't want console (we only tolerate it on frontend but we should move them to sentry log)

Comment (UnbearableBear) [reply .eslintrc.yml]: Hum the issue is that most of our files in the dataset folder use `console.log` to pipe the output into files with `>>`

Comment (lionelB) [reply .eslintrc.yml]: what if we add "root" eslint config in `code-du-travail-data` where we override the use of console ?

Comment (UnbearableBear) [reply .eslintrc.yml]: Makes total sense !

Comment (UnbearableBear) [reply .eslintrc.yml]: done !

**Review by lionelB — APPROVED** [review]

**Pull request merged**